### PR TITLE
PEP 458: Clarify sequence of PyPI and pip integration

### PR DESCRIPTION
**Pull request by brainwane**
OK'd by coauthors in [Discourse conversation](https://discuss.python.org/t/pep-458-secure-pypi-downloads-with-package-signing/2648/132).

### CODE REVIEW

**Review by mnm678 — APPROVED** [review]

**Review by joshuagl — APPROVED** [review]

**Review by hugovk — APPROVED** [review]

**Comment by hugovk** [comment]
This PEP is commented out of the CODEOWNERS file:

https://github.com/python/peps/blob/ca6d20a04aa24e1520b0c4bfbf7f2a0adcaa4855/.github/CODEOWNERS#L318

Probably because the PEP predates the CODEOWNERS file.

[PEP 1](https://peps.python.org/pep-0001/#submitting-a-pep) says:

> Update [.github/CODEOWNERS](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners) such that any co-author(s) or sponsors with write access to the [PEP repository](https://github.com/python/peps) are listed for your new file. This ensures any future pull requests changing the file will be assigned to them.

Shall we add the sponsor, @ncoghlan, to CODEOWNERS for this PEP?

**Comment by CAM-Gerlach** [comment]
That would be SOP, but up to @ncoghlan if he still wants to be sponsor, or maybe better to have @dstufft , the PDP-Delegate, be sponsor instead since he's more active nowadays with regard to PyPI, AFAIK.

**Review by di — APPROVED** [review]

**Comment by hugovk** [comment]
We can update `CODEOWNERS` as and when, let's get this merged. Thanks!

**Pull request merged**
